meteor/sofie-server/model/entities/ad-lib-piece.ts

```diff
@@ -0,0 +1,50 @@
+import { TimelineObject } from './timeline-object'
+
+export interface AdLibPieceInterface {
+	id: string
+	rundownId: string
+	name: string
+	duration: number
+	timelineObjects: TimelineObject[]
+}
+
+export class AdLibPiece {
```

Comment:
4 / 5 of the properties are in common with properties of the `Piece` interface. 
Could some common interfaces possibly be made?

Among those common ones are e.g `id` and `name`, that all the enitiy interfaces seem to have.

Reply:
You already have a interface called `Identifier`, which holds `id` and `name`.
Why not extend that interface?

Reply:
This comment chain should propably have been made at the interface definition. Oh well...

Reply:
We could. I don't have an argument for or against except there is a fine line between reducing duplication and maintaining simplicity/readability.